lib/utils/pyhelp.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Colin Hogben
```

Comment:
> Usually I don't put any copyright header in such cases.

I believe all files in the repo have the same header template, which has copyright statement, including .h files.

> What is your preference for new but trivial code such as pyhelp.h?

It should be the same as original file from where it was split, or just Damien (majority of code in the repo authored by him).